server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/variables/GroupItemNode.java

```diff
@@ -14,39 +14,82 @@
  */
 package org.eclipse.lsp.cobol.core.model.tree.variables;
 
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.ToString;
 import org.eclipse.lsp.cobol.core.messages.MessageTemplate;
 import org.eclipse.lsp.cobol.core.model.Locality;
 import org.eclipse.lsp.cobol.core.model.SyntaxError;
+import org.eclipse.lsp.cobol.core.model.tree.Node;
 import org.eclipse.lsp.cobol.core.model.tree.NodeType;
+import org.eclipse.lsp.cobol.core.model.variables.UsageFormat;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Stream;
 
 import static org.eclipse.lsp.cobol.core.model.tree.variables.VariableDefinitionUtil.EMPTY_STRUCTURE_MSG;
 
 /**
  * This value class represents a group item COBOL variable. Group elements can have nested
  * variables. They cannot have neither PIC nor VALUE clauses. Can be the top element of the
  * structure with level 01, as well as nested ones with higher level numbers.
+ *
+ * <p>Group item can have usage clause.<br>
+ * <br>
+ *
+ * <pre>
+ *  Case 1.
+ *    Working-Storage Section.
+ *    01 FOO POINTER GLOBAL.   <-- This is valid. Here FOO node has
```

Comment:
This line will be parsed as ElementaryItemNode, not the GroupItemNode. You don't need to care about the case here.